EasyRestProjectNetTeam2/EasyRestTests/CheckSignUpTests.cs

```diff
@@ -1,7 +1,117 @@
-﻿namespace EasyRestProjectNetTeam2.EasyRestTests
+﻿using EasyRestProjectNetTeam2.EasyRestPages;
+using NUnit.Framework;
+
+namespace EasyRestProjectNetTeam2.EasyRestTests
 {
+    [TestFixture]
     class CheckSignUpTests : BaseTest
     {
+        HomePage _homePage;
+        SignUpPage _signUpPage;
+
+        [SetUp]
+        public new void  SetUp()
+        {
+            base.SetUp();
+            _homePage = GetHomePage();
+            _homePage.HeaderMenuComponent.ClickSignUpButton();
+            _signUpPage = GetSignUpPage();
+            _signUpPage.ClickCreateAccountButton();
```

Comment:
Please move this line into `CheckUserIsUnableToSignUpWithEmptyFields()` because this step is not common for all tests in that Test Fixture, so it should not be in the `Setup()` method.

Reply:
Resolved